templates/.vscode/launch.json.tpl

```diff
@@ -54,6 +54,22 @@
         }
       ],
     },
+    {
+      "name": "Attach to dev container (in binary mode)",
```

Comment:
Here we don't need _substitutePath_ magic. I haven't found better way than having another debug configuration